api/v2/views/instance_action.py

```diff
@@ -12,11 +12,11 @@ class InstanceActionFilter(filters.FilterSet):
     provider_id = django_filters.CharFilter(method='filter_by_provider')
     instance_id = django_filters.CharFilter(method='filter_by_instance')
 
-    def filter_by_instance(self, queryset, value):
+    def filter_by_instance(self, queryset, query_key, query_value):
```

Comment:
I didn't realize this when I created https://github.com/cyverse/atmosphere/pull/288 but this should have been included in that PR. The arg-names should actually probably be `name` and `value` to avoid confusion. Thanks!